continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/multimodule-api/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name><!-- [INSERT NAME_MODULE_1 HERE] --></name>
+    <description><!-- [INSERT DESCRIPTION_MODULE_1 HERE] --></description>
+
+    <parent>
+        <groupId><!-- [INSERT GROUP_MM HERE] --></groupId>
+        <artifactId><!-- [INSERT ID_MM HERE] --></artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <groupId><!-- [INSERT GROUP_MM HERE] --></groupId>
+    <artifactId><!-- [INSERT MODULE_1 HERE] --></artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!-- VERSIONING -->
```

Comment:
Is there an expected version number to be inserted here?

Reply:
No, that leverages our parent POM and CI versioning. I guess we should mention that.

Reply:
Ah I see - we may want to make regular comments lowercase if UPPERCASE is being used for templating.